pyproject.toml

```diff
@@ -27,24 +27,25 @@ classifiers = [
 
 
 [tool.poetry.dependencies]
-python = "^3.6.1"
-graphenelib = "^1.2.0"
-toolz = "^0.10.0"
-funcy = "^1.14"
-docker = "^4.2.0"
+python = "^3.10.0"
```

Comment:
Generally libraries are trying to maintain compatibility with older python versions. In this case I'm not sure there are much clients to support.